pkg/storage/storage.go

```diff
@@ -0,0 +1,73 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Profile struct {
+	Name string `json:"name"`
+	Key  string `json:"key"`
+}
+
+type Storage struct {
```

Comment:
Constructor is missed? Why do you need to call it as `&Storage{}` and fail on basic initialization while constructor could restrict options needed to initialise and handle bad input?

```
st,err := storage.New("default", databasePath)
```